pkg/scheduler/framework/runtime/framework.go

```diff
@@ -403,6 +423,14 @@ func updatePluginList(pluginList interface{}, pluginSet *config.PluginSet, plugi
 	return nil
 }
 
+// FeatureGates returns whether a feature gate is registered and enabled with the handle
+func (f *frameworkImpl) FeatureGates(feature featuregate.Feature) bool {
```

Comment:
This package is already depending on the `utilfeature` package, so I don't see the need to have this map.

Just call `DefaultFeatureGate.Enabled` here and remove the `With` functions